metadetect/metadetect.py

```diff
@@ -120,6 +146,23 @@ def _add_positions_and_psf(self, cat, res):
 
             newres['sx_col'] = cat['x']
             newres['sx_row'] = cat['y']
+
+            rows_noshear, cols_noshear = shearpos.unshear_positions(
+                newres['sx_row'],
+                newres['sx_col'],
+                shear_str,
+                obs,  # an example for jacobian and image shape
+            )
```

Comment:
Looks like we didn't feed the shear amount here. That must be around somewhere, right?

Reply:
I think we should so this doesn't break later if we change it.

Reply:
we always use 0.01, but I see your point

Reply:
currently we are not passing this around, so I'm gonna punt on it

don't use anything other than 0.01 :)